src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2294,6 +2294,12 @@ const bool Matcher::match_rule_supported(int opcode) {
         ret_value = false;
       }
       break;
+    case Op_ExpandBits:
+    case Op_CompressBits:
+      if (!(UseSVE > 1 && VM_Version::supports_svebitperm())) {
```

Comment:
Do we need to test `UseSVE` here?

Reply:
We don't want to enable this on detection of the feature - if UseSVE is disable, we'll be missing the SVE infrastructure that we are dependent on in the compiler.

Reply:
Correction - we don't want to enable this solely on the present of the feature. With -XX:-UseSVE on the command line, this would break, so we need UseSVE and the feature to be present.

Reply:
My understanding: 
`VM_Version::supports_svebitperm()` means if the hardware supports the feature or not. `UseSVE` means if user wants to use SVE(2).

Reply:
OK, got it.